docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -0,0 +1,391 @@
+---
+id: apifctl-cicd-integration
+title: CI/CD Platform Integration with apifctl
+sidebar_label: CI/CD Integration with apifctl
+description: "Using Sauce Labs API Testing or CLI, you can seamlessly integrate continuous API testing into your CI/CD pipeline."
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Description
+
+Execute API tests and interact with Sauce Labs API Testing (either locally or in a pipeline) using our API Testing CLI tool, `apifctl`.
+
+## Usage
+
+`$ docker run quay.io/saucelabs/apifctl [COMMAND] [OPTIONS]`
+
+## What You'll Need
+* A [Docker](https://docs.docker.com/get-docker/) account.
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* Your Sauce Labs [Username](https://app.saucelabs.com/user-settings) and [Access Key](https://app.saucelabs.com/user-settings).
+
+### Adding Webhooks
+To utilize most `apifctl` functionalities, you'll need to add a webhook to your API Testing Project by either generating one or retrieve one from your Dashboard.
+
+Below is how webhook URLs are formatted, where `<username>` is your Sauce Labs username, `<access_key>` is your Sauce Labs access key, and the `<hookId>` is what you would generate in the API Testing UI.
+```bash
+https://<username>:<access_key>@domain/api-testing/rest/v4/<hookId>
+```
+
+To generate a webhook:
+1. Navigate to your Project and select the **WebHooks** tab.<br/>
+   <img src={useBaseUrl('img/api-fortress/2021/04/webHooksSection.png')} alt="webhook screenshot"/>
+2. Select **Create Hook**.<br/>
+   <img src={useBaseUrl('img/api-fortress/2021/04/createHook.png')} alt="Create New WebHook"/>
+3. Enter the field details and click **Save**.<br/>
+   <img src={useBaseUrl('img/api-fortress/2021/04/sampleHook.png')} alt="sample webhook details" width="500" />
+4. A generated **Hook URL** will then appear. It will also become available in your [Sauce Labs user settings](https://app.saucelabs.com/user-settings). Copy the **Hook URL** to your clipboard and use it either locally or as part of CI build. <br/>
```

Comment:
Remove, it's not true: It will also become available in your [Sauce Labs user settings](https://app.saucelabs.com/user-settings).